frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -676,19 +692,11 @@ class Worksheet extends React.Component {
         if (subIndex === 'end') {
             subIndex = (this._numTableRows(info.blocks[index]) || 1) - 1;
         }
-        if (
-            index < -1 ||
-            index >= info.blocks.length ||
-            subIndex < -1 ||
-            subIndex >= (this._numTableRows(info.blocks[index]) || 1)
-        ) {
-            console.log('Out of bounds');
-            return; // Out of bounds (note index = -1 is okay)
-        }
+
         let focusedBundleUuidList = [];
         if (index !== -1) {
             // index !== -1 means something is selected.
-            // focusedBundleUuidList is a list of uuids of all bundles after the selected bundle (itself included)
+            // focusedBundleUuidList is a list of ids of all items after the selected bundle (itself included)
```

Comment:
Are bundles' ids the same as their uuid? If so, better comment here as well

Reply:
Thanks for your advice, I have included it in the comments.